templates/partials/modal_messagehistory.haml

```diff
@@ -7,7 +7,7 @@
     .message-history
       .action.stackitem{ ng-repeat:"action in actions" }
         .action-time
-          [[ action.created_on | autodate ]]
+          <cp-tooltip display-text="[[ action.created_on | autodate]]" tooltip-text="[[ action.created_on | fulldate ]]" position="top-right"><cp-tooltip>
```

Comment:
I'm not entirely convinced we get value from having a custom directive for tooltips which essentially implements uib-tooltip. What if we had a directive for displaying date values ? Something like:

```
<cp-date value="item.time" />
```

Which does the appropriate formatting and setup of the tooltip.